invoicer_project/invoicer_project/settings.py

```diff
@@ -40,9 +40,11 @@
     'rest_framework',
     'rest_framework_simplejwt',
     'rest_framework_simplejwt.token_blacklist',
+    'reportlab',
```

Comment:
please, add this dependency into `requirements.txt` file

Reply:
Done